CHANGELOG.md

```diff
@@ -10,6 +10,29 @@ Read more details in the issue [#52: Releases and versioning](https://github.com
 
 ## [Unreleased]
 
+## [v1.0.1] 2022-01-24
+
+### Added
+
+- Added a check for data fields and supported location types of the coupling interface [#206](https://github.com/precice/openfoam-adapter/pull/206).
+- Extended the adapter's DEBUG output to print the rank in addition to the message in parallel runs
+[#201](https://github.com/precice/openfoam-adapter/pull/201).
+
+### Changed
+
+- OpenFOAM version bumped to v2112 in GitHub Actions (including preCICE v2.2.1 --> v2.3.0) and documentation. GitHub Action clang-format-action switched to main branch. [#211](https://github.com/precice/openfoam-adapter/pull/211).
+- OpenFOAM version bumped to v2106 in GitHub Actions (including preCICE v2.2.0 --> v2.2.1) and documentation. [#185](https://github.com/precice/openfoam-adapter/pull/185).
+
+### Removed
+
+- Removed explicit casting of boundary conditions in the adapter's write function in order to allow more boundary conditions to be compatible with the adapter (e.g. groovyBC) [#195](https://github.com/precice/openfoam-adapter/pull/195).
```

Comment:
The "Removed" is mainly meant for features. This can stay in the changed:

```suggestion
- Removed explicit casting of boundary conditions in the adapter's write function in order to allow more boundary conditions to be compatible with the adapter (e.g. groovyBC) [#195](https://github.com/precice/openfoam-adapter/pull/195).
```